src/os/view/market/appCard/appCardInfo.tsx

```diff
@@ -51,21 +77,16 @@ const AppCardInfo = ({ appId }: { appId: string }) => {
               </Typography.Text>
             </Space>
           </Col>
-          <Col>
-            {appIds.includes(appId) ? (
-              <Button
-                type="ghost"
-                size="small"
-                onClick={(e) => onOpen(e, appId)}
-              >
-                Open
-              </Button>
-            ) : (
-              <Button type="primary" onClick={onInstall} size="small">
-                Install
-              </Button>
-            )}
-          </Col>
+          {account.isAddress(address) && (
+            <Col>
+              <ActionButton
+                appIds={appIds}
+                appId={appId}
+                onOpen={(e) => onOpen(e)}
```

Comment:
`onOpen={(e) => onOpen(e)}` to `onOpen={onOpen}` is good enough. Similarly to `onInstall`